assets/section-main-product.css

```diff
@@ -240,6 +240,7 @@ fieldset.product-form__input .form__label {
 
 .product-form__input .select {
   max-width: 25rem;
+  padding-left: var(--inputs-border-width);
 }
```

Comment:
On product selects, the styling on hover seems off: [video](https://screenshot.click/22-01-9m3ew-q0xju.mp4)